lib/modules/subredditInfo.js

```diff
@@ -61,7 +61,8 @@ modules['subredditInfo'] = {
 	srRe: /\/r\/(\w+)(?:\/(new|rising|controversial|top))?\/?$/i,
 	addListeners: function(ele) {
 		ele = ele || document.body;
-		var subredditLinks = document.body.querySelectorAll('.listing-page a.subreddit, .explore-page a.subreddit, .comment .md a[href^="/r/"], .side a[href^="/r/"], .usertext-body a[href^="/r/"], .trending-subreddits a[href^="/r/"]');
+        var querySelectors = ['.listing-page a.subreddit','.explore-page a.subreddit','.comment .md a[href^="/r/"]','.comment .md a[href*="reddit.com/r/"]','.side a[href^="/r/"]','.side a[href*="reddit.com/r/"]','.usertext-body a[href^="/r/"]','.usertext-body a[href*="reddit.com/r/"]','.trending-subreddits a[href^="/r/"]','.trending-subreddits a[href*="reddit.com/r/"]'];
```

Comment:
or we could go the other direction, which is probably bad performance in a different way:

    $(document.body).on('mouseover', 'a[href*="/r/"]', 
        function(e) { if (this.href.test(RESUtils.subredditRegex)) {
          hover blah blah
    });